server/src/main/scala/org/scassandra/server/cqlmessages/response/Response.scala

```diff
@@ -28,7 +30,7 @@ abstract class Response(header : Header) extends CqlMessage(header)
 
 case class Ready(stream : Byte = ResponseHeader.DefaultStreamId)(implicit protocolVersion: ProtocolVersion) extends Response(new Header(protocolVersion.serverCode, opCode = OpCodes.Ready, streamId = stream)) {
 
-  implicit val byteOrder = java.nio.ByteOrder.BIG_ENDIAN
+  implicit val byteOrder : ByteOrder = CqlProtocolHelper.byteOrder
```

Comment:
It looks like I missed this one, I had fixed in other places but must have missed here.